Source/WebCore/platform/graphics/NativeImage.h

```diff
@@ -45,6 +45,8 @@ class NativeImage final : public RenderingResource {
     WTF_MAKE_FAST_ALLOCATED;
 public:
     static WEBCORE_EXPORT RefPtr<NativeImage> create(PlatformImagePtr&&, RenderingResourceIdentifier = RenderingResourceIdentifier::generate());
+    // Creates a NativeImage that will not be used many times. Avoids putting the image to any caches.
```

Comment:
Slightly clearer phrasing:

`// Creates a NativeImage that will not be used frequently. Avoids storing the image in any caches.`

Reply:
Right, but frequently and many times are different.
You can get 100 uses infrequently vs. you can get only 1 use. Then once per 1ms is frequently for somebody, and for other it might be infrequent.. I can rephrase..
.